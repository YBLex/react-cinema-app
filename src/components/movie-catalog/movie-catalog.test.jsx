import React from 'react';
import renderer from 'react-test-renderer';

import MovieCatalog from './movie-catalog.jsx';

const Mock = () => <div className="mock"></div>;

it(`Render MovieCatalog`, () => {
  const tree = renderer.create(
      <MovieCatalog>
        <Mock />
      </MovieCatalog>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
