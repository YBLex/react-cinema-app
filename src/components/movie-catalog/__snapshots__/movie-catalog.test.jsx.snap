// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieCatalog 1`] = `
<section
  className="catalog"
>
  <h2
    className="catalog__title visually-hidden"
  >
    Catalog
  </h2>
  <div
    className="mock"
  />
</section>
`;
